File: PREPARCIAL/Python/trigonometria.py
from easyinput import read
import math

def main():
    angle = read(float)
    while angle is not None:
        print ("{:.6f}".format( math.sin(angle * (math.pi / 180))), "{:.6f}".format (math.cos(angle * (math.pi / 180))))
        angle = read(float)
main()

File: PREPARCIAL/Seqüències/trigonometria.cc
#include <cmath>
#include <iostream>
using namespace std;

int main()
{
    cout.setf(ios::fixed);
    cout.precision(6);

    double angle;
    while (cin >> angle) {
        double anglerad = angle * (M_PI / 180);
        cout << sin(anglerad) << " "
             << cos(anglerad) << endl;
    }
}
